pytomorrowio/pytomorrowio.py

```diff
@@ -198,15 +194,10 @@ async def realtime(
                 "fields": fields,
             }
         )
-        if (
-            "data" in data
-            and "timelines" in data["data"]
-            and "intervals" in data["data"]["timelines"][0]
-            and "values" in data["data"]["timelines"][0]["intervals"][0]
-        ):
+        try:
             return data["data"]["timelines"][0]["intervals"][0]["values"]
-
-        return {}
+        except (IndexError, KeyError) as error:
```

Comment:
FWIW, IndexError and KeyError are both subclasses of LookupError.

Reply:
good to know!